one_shot_scripts/update_deck_format.js

```diff
@@ -193,11 +193,24 @@ async function buildDeck(cards, bot) {
   }
 }
 
-async function update(deck) {
-  if (deck.seats && deck.seats.length > 0) {
-    return Deck.updateOne({ _id: deck._id }, deck);
+// this just forms lands correctly
+function convertDeckCard(card) {
+  if (Array.isArray(card)) {
+    return card.map(convertDeckCard);
+  }
+  if (card.cardID) {
+    return card;
   }
+  return {
+    tags: [],
+    colors: card.details.colors,
```

Comment:
```suggestion
    colors: card.details.color_identity,
```

Reply:
idk if this matters but it's correct

Reply:
for this case, it's fine -  these colors aren't used for any analytics or anything